.travis.yml

```diff
@@ -45,12 +46,12 @@ jobs:
       install: .travis/install-debian.sh
       script: .travis/build-debian.sh
     - stage: test
-      name: "Windows Visual Studio 2017 test"
+      name: "Windows test"
```

Comment:
Why do we annotate jobs now with their stages? That's a little redundant, isn't it?

Reply:
True, I updated it, will force push it on here